penn/wharton.py

```diff
@@ -3,6 +3,7 @@
 
 from bs4 import BeautifulSoup
 from .base import APIError
+from flask import jsonify, request
```

Comment:
You don't want to reference flask at all in `penn-sdk-python`. Ideally, `penn-sdk-python` should be able to work by itself, in situations where flask doesn't exist or the code isn't being used in a web server.

If you need any variables, pass them through as arguments in your function.